scripts/build1.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
Improved documentation would be preferred over adding another script, as this is already attainable with the current build scripts.